src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBERS_LIMIT = 100;
+    private final ColorSupplier colorBall = new ColorSupplier();
+    private final Random numberBall = new Random();

```

Comment:
```suggestion
    private final ColorSupplier color = new ColorSupplier();
    private final Random random = new Random();

```

Reply:
I agree with Oleksiy, it's not an idiomatic naming, we should simplify variables' names. When you read `colorBall` it's confusing for the reader, who may assume it's some kind of ball, but in fact this is a colorSupplier intance
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/3d314074-d803-459e-94cc-7b670b5c9cca)